<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>R-ABLE</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/plugins/r_able/icon.png</normaloff>:/plugins/r_able/icon.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="user_label">
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="user_input"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="password_label">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="password_input">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="name_label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="name_input"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="uri_label">
     <property name="text">
      <string>URI</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="uri_input"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name_input</tabstop>
  <tabstop>uri_input</tabstop>
  <tabstop>user_input</tabstop>
  <tabstop>password_input</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
